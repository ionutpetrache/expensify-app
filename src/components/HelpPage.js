import React from 'react';

const HelpPage = () => (
    <div>HELP page</div>
);

export default HelpPage;