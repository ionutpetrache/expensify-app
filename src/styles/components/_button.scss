.button {
    color: white;
    background-color: $blue;
    border: none;
    display: inline-block;
    padding: 16px;
    border-radius: 2px;
    font-weight: 300;
    font-size: $font-size-large;
    text-decoration: none;
    line-height: 1;

    &--link {
        background-color: transparent;
    }

    &--secondary {
        background-color: #888888;
    }
}