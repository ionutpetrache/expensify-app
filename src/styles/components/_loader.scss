.loader {
    display: flex;
    justify-content: center;
    align-items: center;
    height: 100vh;
    widows: 10vw;
}

.loader__image {
    height: 6rem;
    width: 6rem;
}