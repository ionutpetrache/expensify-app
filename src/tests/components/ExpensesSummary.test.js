import React from 'react';
import {shallow} from 'enzyme';
import {ExpensesSummary} from '../../components/ExpensesSummary';

test('should correctly render ExpensesSummaru with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={100} />);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render ExpensesSummaru with multiple expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={2450} />);
    expect(wrapper).toMatchSnapshot();
});
