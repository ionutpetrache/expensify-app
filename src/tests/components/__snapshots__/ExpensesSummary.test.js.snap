// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpensesSummaru with 1 expense 1`] = `
<div>
  <div>
    Viewing 
    1
     
    expense
     totalling: $
    100
  </div>
</div>
`;

exports[`should correctly render ExpensesSummaru with multiple expense 1`] = `
<div>
  <div>
    Viewing 
    23
     
    expenses
     totalling: $
    2450
  </div>
</div>
`;
